app/src/main/java/com/ciandt/dragonfly/example/features/download/DownloadHelper.kt

```diff
@@ -21,11 +21,16 @@ object DownloadHelper {
         ProjectRepository(database)
     }
 
-    fun startDownload(context: Context, title: String, version: Version, uri: Uri) = runOnBackgroundThread {
+    fun startDownload(context: Context, title: String, description: String?, version: Version, uri: Uri) = runOnBackgroundThread {
         val request = DownloadManager.Request(uri)
 
-        val appName = context.getString(R.string.app_name)
-        request.setTitle("$appName: $title")
+        val downloadTitle = context.getString(R.string.download_title, context.getString(R.string.app_name), title)
```

Comment:
Can't we define download_title as something like:
<string name="download_title">@string/app_name: %1$s</string>
instead?

Feels like we won't never have a value other than app name for this parameter, right?